Code/Framework/AzCore/Tests/AZStd/RangesViewTests.cpp

```diff
@@ -650,7 +651,20 @@ namespace UnitTest
         static constexpr auto arrayOfLiterals{ AZStd::to_array<AZStd::string_view>({"Hello", "World", "Moon", "Sun"}) };
         auto commonView = AZStd::ranges::views::common(AZStd::ranges::views::join_with(arrayOfLiterals, ','));
 
-        AZStd::fixed_string<128> accumString{commonView.begin(), commonView.end()};
+        AZStd::fixed_string<128> accumString{ commonView.begin(), commonView.end() };
         EXPECT_EQ(expectedString, accumString);
     }
+
+    TEST_F(RangesViewTestFixture, FilterView_CanFilterWhiteSpaceFromString_Succeeds)
+    {
```

Comment:
question: This might be fine but it seems like there could be some other tests we could write to exercise all the functionality added with `filter_view`. It might be worth trying to generate some coverage (we have some info on using OpenCppCoverage I can share with you) to see what functions are covered and which aren't.

Reply:
I don't believe in utility of code coverage tools, that being said.
I do believe in writing in test that invokes all import public functions, so I will add some test.

Reply:
Okay sure thing, it's hard to know what is covered or not and I know arbitrary coverage numbers aren't useful but it can be useful in this case to know what we might be missing. Not critical just worth seeing if any other important public functions (that could be used in interesting ways) aren't covered yet. Cheers for checking 👍

Reply:
Well I wrote the filter_view to use it in a scenario in the Settings Registry Origin Tracker and wrote a quick test to validate teh scenario I was using it in, but I should have added more test before putting it up for PR